Structure_Engine/Query/Area.cs

```diff
@@ -45,7 +45,7 @@ public static partial class Query
         [Output("area", "The area of the FEMesh.", typeof(Area))]
         public static double Area(this FEMesh mesh)
         {
-            return Engine.Analytical.Query.Geometry(mesh).Area();
+            return mesh.IsNull() ? 0 : Analytical.Query.Geometry(mesh).Area();
```

Comment:
I am happy with either, although if it does return 0 you will still get an error message telling you what is null - so it's not like the user will be completely unaware.

Reply:
> I am happy with either, although if it does return 0 you will still get an error message telling you what is null - so it's not like the user will be completely unaware.

Very good point. That makes it feel more comfortable returning 0 rather than `NaN`.

Still happy for more opinions though!

Reply:
Agreed with returning 0.

Summarising and I think reinforcing points made by @peterjamesnugent and @IsakNaslundBh above - returning `double.NaN` is not additionally helpful here (in fact it will simply make more problems downstream potententally forcing further filtering and data clean up of `NaN`s in operations on lists of doubles). 

It is the incoming object that  is `null` and that is already  being flagged and error thrown in `.IsNull()` method. 

A case as to whether `0` or `NaN` is the "right" answer can easily be made either way I think - so keeping primitives "simple" and avoiding `NaN`s would be my preference.

And so, wrt query methods returning primitives, returning 0  is best I think.